encoding/ipfix/decode.go

```diff
@@ -110,7 +109,7 @@ func readTemplateSet(reader *bytes.Reader, out *TemplateSet) (int64, error) {
 		bytesRead += templateRecordHeaderLength
 
 		for i := uint16(0); i < templateRecord.TemplateHeader.FieldCount; i++ {
-			fieldSpecifier := FieldSpecifier{0, 0, 0}
+			fieldSpecifier := fieldSpecifier{0, 0, 0}
```

Comment:
Codacy found an issue: [avoid magic numbers like '0', create a named constant for it](https://app.codacy.com/gh/mateumann/clouseau/pullRequest?prid=6383124)